apidoc/Titanium/Android/Android.yml

```diff
@@ -108,10 +108,10 @@ description: |
     Starting from Android 6.0 (API level 23), users need to grant certain permissions to apps while the
     app is running. You can read it more [here](https://developer.android.com/training/permissions/requesting.html).
 
-    In Titanium SDK, to support this, we have the <Titanium.Android.requestPermissions> method. It is used to 
+    In Titanium SDK, to support this, we have the <Titanium.Android.requestPermissions> method. It is used to
     request any permission you may need. An example of using it is shown below:
 
-        var permissions = [ 'android.permission.CAMERA', 'android.permission.READ_EXTERNAL_STORAGE' ];
+        var permissions = [ 'android.permission.CAMERA', 'android.permission.READ_EXTERNAL_STORAGE', 'android.permission.READ_EXTERNAL_STORAGE' ];
```

Comment:
`WRITE_EXTERNAL_STORAGE`

Reply:
one small change and still I made a Copy&paste error 🙈 Thanks